### PR TITLE
enhance: support bitmap mmap

### PR DESCRIPTION
**Pull request by zhagnlu**
#32900

### CODE REVIEW

**Comment by zhagnlu** [comment]
rerun ut

**Comment by zhagnlu** [comment]
rerun ut

File: pkg/util/indexparamcheck/index_type.go [hunk 1]
```diff
@@ -70,7 +70,9 @@ func IsVectorMmapIndex(indexType IndexType) bool {
 		indexType == IndexHNSW ||
 		indexType == IndexScaNN ||
 		indexType == IndexSparseInverted ||
-		indexType == IndexSparseWand
+		indexType == IndexSparseWand ||
+		indexType == IndexBitmap ||
+		indexType == IndexHybrid
```
Comment (czs007) [inline pkg/util/indexparamcheck/index_type.go:75]: HybridINdex is not Vector Index

Comment (zhagnlu) [reply pkg/util/indexparamcheck/index_type.go:75]: good catch

**Comment by zhagnlu** [comment]
rerun ut

**Comment by zhagnlu** [comment]
rerun ut

**Pull request merged**